src/views/jobs/moderator/moderator.jsx

```diff
@@ -0,0 +1,94 @@
+var React = require('react');
+var render = require('../../lib/render.jsx');
```

Comment:
@thisandagain Nope, I have a Chromebook, so I don't have the capabilities to do so. (And I still don't have a large enough SD card for my Raspberry Pi ☹️) Sorry about that, I'll fix it.